views.py

```diff
@@ -100,14 +106,16 @@ def post(self):
     if 'max_groups' in solr_args:
         del solr_args['max_groups']
 
-    solr_args["rows"] = min(int(solr_args.get("rows", [current_app.config.get("VIS_SERVICE_PN_MAX_RECORDS")])[0]), current_app.config.get("VIS_SERVICE_PN_MAX_RECORDS"))
+    data = solr_args["q"]
 
+    solr_args["rows"] = min(int(solr_args.get("rows", [current_app.config.get("VIS_SERVICE_PN_MAX_RECORDS")])[0]), current_app.config.get("VIS_SERVICE_PN_MAX_RECORDS"))
     solr_args['fl'] = ['bibcode,title,first_author,year,citation_count,read_count,reference']
     solr_args['wt'] ='json'
+    solr_args['q'] = '*:*'
 
     headers = {'X-Forwarded-Authorization' : request.headers.get('Authorization')}
 
-    response = client().get(current_app.config.get("VIS_SERVICE_SOLR_PATH") , params = solr_args, headers=headers)
+    response = client().post(current_app.config.get(VIS_SERVICE_BIGQUERY_PATH), params = solr_args, headers=headers, data=data)
```

Comment:
you will have to add few parameters (for security sake), the bigquery works as:

```
requests.post('http://localhost:5000/bigquery', params={'q':'*:*', 'wt':'json', 'fl':'bibcode', 'fq': '{!bitset}'}, data='bibcode\n1907AN....174...59.\n1908PA.....16..445.\n1989LNP...334..242S')
```

so, the 'fq' should be set; and I think you should check that data contains \n delimited list of bibcodes only (not bibcode:xxxxx, but xxxxx)